### DISCUSSION TITLE
'multiclass_classification' is not a valid ModelMode

### DISCUSSION THREAD

**Discussion by WHYz919** [pyg-team/pytorch_geometric#6674] [General]
I try to implement the task of map classification in GNNExplainer, but the above error will be reported. My configuration is pytorch1.11.0+cuda11.3.

**Comment by rusty1s** [comment]
The interface changed slightly in current PyG master. You can either upgrade to PyG master, or use the interface from PyG 2.2, see https://github.com/pyg-team/pytorch_geometric/blob/2.2.0/examples/gnn_explainer.py.
